### ISSUE TITLE
Arrays show a redundant title

### ISSUE THREAD

**Issue by awalsh-broadsoft** [swagger-api/swagger-ui#3999]
| Q                               | A
| ------------------------------- | -------
| Bug or feature request?         |  Bug
| Which Swagger/OpenAPI version?  | 3.0.0
| Which Swagger-UI version?       | 3.5.0
| How did you install Swagger-UI? | Tomcat
| Which browser & version?        | Chrome
| Which operating system?         | Windows 10 Pro

### Demonstration API definition
```json
...
    "assignedUserIds": {
      "title": "List of Assigned User Ids",
      "description": "Is a list of a assigned user ids.",
      "type": "array",
      "items": {
        "title": "Individual User Id",
        "description": "User Id",
        "type": "string"
      }
    }
...
```

### Expected Behavior
When displaying an array, I would expect the title to appear only once.

### Current Behavior
The array title appears twice, once in front of the opening square bracket and again just after the square bracket on the next line.
![swagger-array-issue](https://user-images.githubusercontent.com/17168790/33681119-3b1abe26-da89-11e7-9690-43b00b845d0d.PNG)

### Possible Solution
Only show the first title.
Additionally, it seems a bit confusing to have the information about the array as a whole appear inside of the array. Not sure exactly what to do about that. The behavior of having the array information inside of the brackets is behavior I can live with though. The double title does seem redundant.

**Comment by robert-hebel-sb** [comment]
We are doing some housekeeping on our issue tracker. This issue has been inactive for over a year and we have not been able to tackle it yet.

Is this still important to you? If so, please leave a comment – we'd love to know if you're still affected. If we don't hear back in 7 days, we'll close this issue to tidy up the tracker.

Thank you for your understanding!

**Comment by robert-hebel-sb** [comment]
No response received after 7 days. Closing this issue to keep the tracker tidy. If you're still affected, please reopen or create a new issue with updated details.
